forge-gui/res/cardsfolder/upcoming/mizzix_replica_rider.txt

```diff
@@ -0,0 +1,16 @@
+Name:Mizzix, Replica Rider
+ManaCost:4 R
+Types:Legendary Creature Goblin Wizard
+PT:4/5
+K:Flying
+T:Mode$ SpellCast | ValidCard$ Card.wasNotCastFromYourHand | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigCopy | TriggerDescription$ Whenever you cast a spell from anywhere other than your hand, you may pay {1}{U/R}. If you do, copy that spell and you may choose new targets for the copy. If the copy is a permanent spell, it gains haste and "At the beginning of the end step, sacrifice this permanent."
+SVar:TrigCopy:AB$ CopySpellAbility | Cost$ 1 UR | Defined$ TriggeredSpellAbility | MayChooseTarget$ True | RememberNewCard$ True | SubAbility$ DBAnimate
+SVar:DBAnimate:DB$ Animate | Defined$ Remembered | ConditionDefined$ Remembered | ConditionPresent$ Permanent | Keywords$ Haste | Triggers$ EOTSac | Duration$ Permanent
+SVar:EOTSac:Mode$ Phase | Phase$ End of Turn | Execute$ TrigSac | TriggerDescription$ At the beginning of the end step, sacrifice this permanent.
+SVar:TrigSac:DB$ Sacrifice | Defined$ Remembered | SubAbility$ DBCleanup
```

Comment:
- Defined on cards doesn't exist Sacrifice

- Cleanup happens on the wrong card

Reply:
Oh is this a recent change ? I guess just removing the defined should be fine ?

Reply:
I think this was an old mistake that grew over time